process/datadog_checks/process/process.py

```diff
@@ -7,9 +7,18 @@
 from collections import defaultdict
 import time
 import os
+import subprocess
+import sys
 # 3p
 import psutil
 
+
+# Main entry point is meant for checks needing privilege escalation with sudo
```

Comment:
The main entry point needs to be moved after the import statements (see Travis CI check)

Reply:
Indeed, fixed.